sw/device/lib/testing/test_rom/test_rom_start.S

```diff
@@ -177,22 +177,32 @@ _start:
   beqz t0, .L_ast_done_loop
 
 .L_ast_init_skip:
-  // Enable entropy complex - this is not the full enable
-  li   a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
-  li   t0, 0x555055
-  sw   t0, ENTROPY_SRC_CONF_REG_OFFSET(a0)
-
-  li   a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
-  li   t0, 0xa
-  sw   t0, ENTROPY_SRC_MODULE_ENABLE_REG_OFFSET(a0)
-
-  li   a0, TOP_EARLGREY_CSRNG_BASE_ADDR
-  li   t0, 0xaaa
-  sw   t0, CSRNG_CTRL_REG_OFFSET(a0)
-
-  li   a0, TOP_EARLGREY_EDN0_BASE_ADDR
-  li   t0, 0x55aa
-  sw   t0, EDN_CTRL_REG_OFFSET(a0)
+  // The following sequence enables the minimum level of entropy required to
+  // initialize memory scrambling, as well as the entropy distribution network.
+  li a0, TOP_EARLGREY_ENTROPY_SRC_BASE_ADDR
+  // Note for BOOT_ROM initialization the FIPS_ENABLE bit is set to MuBi4False
+  // to prevent the release of FIPS entropy until all the thresholds are set
+  li t0, (0x9 << ENTROPY_SRC_CONF_FIPS_ENABLE_OFFSET) | \
+         (0x9 << ENTROPY_SRC_CONF_ENTROPY_DATA_REG_ENABLE_OFFSET) | \
+         (0x9 << ENTROPY_SRC_CONF_THRESHOLD_SCOPE_OFFSET) | \
+         (0x9 << ENTROPY_SRC_CONF_RNG_BIT_ENABLE_OFFSET)
+  sw t0, ENTROPY_SRC_CONF_REG_OFFSET(a0)
```

Comment:
@alphan @timothytrippel I had to change the mubi values in the mask and test ROM's also, and aligned the two code sections since they were doing the same thing. Can you take a look? Specifically, this `ENTROPY_SRC_CONF_REG_OFFSET` register appeared to have a different value (`0x555055` instead of `0x5555`) in the test ROM and I wanted to double check whether that was an error or whether that was intentional.

Reply:
For refernce:
`kMultiBitBool4True`: 0xA -> 0x6
`kMultiBitBool4False`: 0x5 -> 0x9

Reply:
probably was an error.

Reply:
I agree, looks like an error, or could be due to a change in the register layout.